pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -330,10 +330,10 @@ function validateParams(vmParams) {
             break;
         case URL_SOURCE:
         default:
-            if (!vmParams.source.startsWith("https") &&
+            if (!vmParams.source.startsWith("http") &&
```

Comment:
This doesn't look right -- doesn't this now prevent https:// sources again? we should surely allow these?

Reply:
No, it just says that if the prefix is not http nor ftp nor nfs then show error message. https has the http substring so it will work fine. We never disallowed https in fact.